forest/src/cli/mod.rs

```diff
@@ -175,6 +176,13 @@ pub struct CliOpts {
         possible_values = &["mainnet", "calibnet"],
     )]
     pub chain: String,
+    // env_logger-0.7 can only redirect to stderr or stdout. Version 0.9 can redirect to a file.
+    // However, we cannot upgrade to version 0.9 because pretty_env_logger depends on version 0.7
```

Comment:
We could use `pretty-env-logger` from Github repo not crates.io, or am I missing something?

Reply:
That is an option. I have no strong opinions about whether we do this now or hold off until later when a release is out.

Reply:
If we hold off, let's create a separate issue and get rid of the commented out code (and copy the snippet into the issue).